docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -289,7 +328,22 @@ When you run the `create_kafka_` module as part of the playbook at the end of th
 ----
 . Enter values for the `name`, `short description`, and `openshift_offline_token` fields.
 +
-When you run the `create_service_account` module as part of the playbook at the end of this procedure, Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields in the terminal after it creates the service account.
+When you run the `create_service_account` module as part of the playbook at the end of these steps, Ansible populates the generated service account credentials in the `client_id` and `client_secret` fields in the terminal after it creates the service account.
+
+[id="proc-creating-acl-permissions-ansible_{context}"]
+==== Creating Access Control List permissions
+
+[role="_abstract"]
+After you create a service account to connect to a Kafka instance, you must also set the appropriate level of access for that new account in an Access Control List (ACL) for the Kafka instance.
+
+.Prerequisites
+
+* You have an offline token that authenticates the Terraform resources with the {product-long-rhoas} API.
```

Comment:
Think you're referring to Terraform by mistake here.

Reply:
Yes- good catch!